src/pages/Curation/components/Accordion/index.tsx

```diff
@@ -1,9 +1,12 @@
 import { useState, useCallback, useEffect, useMemo, memo } from 'react';
 
-import PresentLoader from '@/assets/icons/present-loader.png';
+import { useNavigate, useLocation } from 'react-router-dom';
+
+import { recommendPresents } from '@/api/recommend';
 import ActiveUpArrow from '@/assets/svgs/active-up-arrow.svg';
 import DownArrow from '@/assets/svgs/down-arrow.svg';
 
+import giftMove from '../../../../assets/giftMOVE.mp4';
```

Comment:
alias를 사용할 수 있을 것 같습니다!

```suggestion
import giftMove from '@/assets/giftMOVE.mp4';
```